swiftUIHW/swiftUIHW/ContentView.swift

```diff
@@ -8,17 +8,69 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let firstButtonText = "Alert простейший 1"
+        static let firstAlertTitleText = "Показать?"
+        static let firstAlertDefaultText = "Да"
+        static let firstAlertDestructiveText = "НЕТ!"
+        static let secondButtonText = "Alert активный 2"
+        static let secondAlertTitleText = "Показать"
+        static let successText = "Успех"
+        static let thirdButtonText = "ActionSheet простой 3"
+        static let thirdActionSheetTitleText = "Показать?"
+        static let forthButtonText = "Активный ActionSheet 4"
+        static let forthActionSheetWarningText = "Важное объявление!"
+        static let forthActionSheetMessageText = "Вы готовы показать?"
+        static let forthActionSheetDefaultText = "Показать"
+        static let forthActionSheetDefaultActionText = "Ура!!! Нам показали"
+        static let forthActionSheetCancelText = "Отмена, боюсь"
+        static let forthActionSheetCancelActionText = "Нам не показали((("
+    }
+    
+    @State var isError = false
+    @State var isThird = false
+    @State var isSecond = false
+    @State var isFirst = false
```

Comment:
```suggestion
    @State private var isErrorShown = false
    @State private var isThirdAlertShown = false
    @State private var isSecondAlertShown = false
    @State private var isFirstShown = false
```
 если флаг что-то показывает, то пишем что вызывается, и на конце Shown